ts/components/wallet/card/RotatedCards.tsx

```diff
@@ -61,8 +67,29 @@ export class RotatedCards extends React.PureComponent<Props> {
     );
   }
 
+  private cardPreview(wallet: Wallet, isLastItem: boolean): React.ReactNode {
+    const { onClick } = this.props;
+    return fromNullable(wallet).fold(undefined, w => (
+      <>
+        <TouchableDefaultOpacity
+          key={`wallet_${w.idWallet}`}
```

Comment:
Using the key on a child node of a repeated component raises the current warning:

<img src="https://user-images.githubusercontent.com/3959405/93220670-1d1a9a00-f76d-11ea-81e6-70dc76926fe8.png" height="400"/>

this solution should solve this warning:

```javascript
      <React.Fragment key={`wallet_${w.idWallet}`}>
        <TouchableDefaultOpacity
          onPress={() => onClick(w)}
          accessible={true}
          accessibilityLabel={I18n.t("wallet.accessibility.cardsPreview")}
          accessibilityRole={"button"}
        >
          {Platform.OS === "android" && <View style={styles.shadowBox} />}
          <View style={styles.rotatedCard}>
            <CardComponent type={"Preview"} wallet={w} />
          </View>
        </TouchableDefaultOpacity>
        {!isLastItem && <View spacer={true} />}
      </React.Fragment>
```